.github/workflows/linux_sanitizers.yml

```diff
@@ -36,14 +36,3 @@ jobs:
       run: |
           cmake --build build --target all
           cmake --build build --target examples
-    - name: Test
-      shell: bash
-      run: |
-          export ASAN_OPTIONS=fast_unwind_on_malloc=0
-          export LSAN_OPTIONS=suppressions=$PWD/tools/asan.supp
-          export UBSAN_OPTIONS=print_stacktrace=1,suppressions=$PWD/tools/ubsan.supp
-          cd build
-          ctest \
-            --output-on-failure \
-            --tests-regex tests.examples \
-            --exclude-regex tests.examples.transpose.transpose_block_numa
```

Comment:
We definitely want to keep these since the sanitizer checks are done at runtime!